amanuensis/resources/userdatamodel/userdatamodel_message.py

```diff
@@ -1,57 +0,0 @@
-from flask import current_app
-from sqlalchemy import func
-from cdislogging import get_logger
-from amanuensis.errors import NotFound, UserError
-from amanuensis.models import (
-    Message,
-)
-
-# do this until AWS-related requests is handled by it's own project
-from pcdcutils.environment import is_env_enabled
-
-logger = get_logger(__name__)
-
-__all__ = [
-    "get_all_messages",
-    "get_messages_by_request",
-    "send_message",
-]
-
-
-def get_all_messages(current_session, logged_user_id):
-    messages = current_session.query(Message).filter_by(sender_id=logged_user_id).all()
-    return messages
-
-
-def get_messages_by_request(current_session, logged_user_id, request_id):
-    # messages = current_session.query(Message).filter(Message.sender_id == logged_user_id, Message.request_id == request_id).all()
-    messages = (
-        current_session.query(Message).filter(Message.request_id == request_id).all()
-    )
-    return messages
-
-
-def send_message(
-    current_session, logged_user_id, request_id, subject, body, receivers, emails
-):
-    """store message in db and send message to emails via aws ses"""
-    new_message = Message(sender_id=logged_user_id, body=body, request_id=request_id)
-    if receivers:
-        new_message.receivers.extend(receivers)
```

Comment:
did you decide to not reimplement this file because it is not used currently? Ad we can back to it and create it from zero when time comes?